### ISSUE TITLE
Loss terms missing

### ISSUE THREAD

**Issue by eunzihong** [kimren227/ExtrudeNet#4]
Hi,

Thank you for your great work on shape parsing and CAD reconstruction of 3D point cloud. 
I and my colleagues recently came across your paper and got really inspired by it, and trying to build upon your brilliant work.

I'm trying to train a model with preprocessed chair dataset with the given chair config file, however the key error of loss dictionary happens during the training.
In the training code, 'Loss_Drift', 'Loss_Support', 'Loss_Control_Polygon' are supposed to be included in the loss dictionary. But the Loss class in loss.py doesn't have these losses.
So I'm running the code without these losses by commenting them out, but I'm wondering that it's okay.

If I'm missing something here, please let me know.

Thank you in advance.
Eunji.
